projects/kit/src/lib/utils/time/parse-time-string.ts

```diff
@@ -14,11 +14,14 @@ export function parseTimeString(
     timeString: string,
     timeMode: MaskitoTimeMode,
 ): Partial<MaskitoTimeSegments> {
+    const isAM = timeString.includes('AM');
+    const isPM = timeString.includes('PM');
```

Comment:
No need to create variable if it used only once and don't increase readability.

```ts
const isAM = timeString.includes('AM');
const isPM = timeString.includes('PM');

// [...]

if (isPM && ...) {
    // [...]
}

if (isAM && ...) {
    // [...]
}
```

is equally readable as 
```ts
if (timeString.includes('AM') && ...) {
    // [...]
}

if (timeString.includes('PM') && ...) {
    // [...]
}
```

Reply:
changes applied